olp-cpp-sdk-authentication/src/AuthenticationClient.cpp

```diff
@@ -347,8 +323,14 @@ client::CancellationToken AuthenticationClient::Impl::SignInClient(
             break;
         }
       });
-  return client::CancellationToken(
-      [&network, request_id]() { network.Cancel(request_id); });
+  auto request_id = send_outcome.GetRequestId();
```

Comment:
it is not given that you always will have an positive outcome.
See https://github.com/heremaps/here-olp-edge-sdk-cpp/pull/96/files#diff-bbc42a63ded02006254f767d3dc2cd3bR72 on how to handle correctly

Reply:
Do you refer ln107 and on?

Reply:
Yes, seems line number mismatch. Theoretically you need to check the outcome:

```cpp
if (!send_outcome.IsSuccessfull()) {
    // TODO: Get error and create error result
    callback(result);
  }

   auto request_id = send_outcome.GetRequestId();
```

Reply:
done